src/parser/jobs/pld/index.tsx

```diff
@@ -1,19 +1,20 @@
-import {t} from '@lingui/macro'
-import TransMarkdown from 'components/ui/TransMarkdown'
+import {Trans} from '@lingui/macro'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
 import CONTRIBUTORS, {ROLES} from 'data/CONTRIBUTORS'
 import {Meta} from 'parser/core/Meta'
 import React from 'react'
 
-const description = t('pld.about.description')`As the illegitimate child of a WHM and BLM, you chose the Gory Path of a shield lobing, sword swinging Mage, that also tries to help everybody out.
-
-This analyzer attempts to find just the right things to get you to be a fearsome Tank, that will show no [~action/CLEMENCY] to his enemies, while being the true [~action/HOLY_SPIRIT] of the Party.
-`
-
 export default new Meta({
 	modules: () => import('./modules' /* webpackChunkName: "jobs-pld" */),
 
 	Description: () => <>
-		<TransMarkdown source={description} key="pld.about.description"/>
+		<Trans id="pld.about.description">
+			<p>As the illegitimate child of a WHM and BLM, you chose the gory path of a shield lobbing, sword swinging Mage that also tries to help everybody out.
+			</p>
+			<p>This analyzer attempts to find just the right things to get you to be a fearsome tank that will show no <ActionLink {...ACTIONS.CLEMENCY}/> to your enemies, while being the true <ActionLink {...ACTIONS.HOLY_SPIRIT}/> of the party.
```

Comment:
inb4 ack kills me, did you mean `analyser`? I feel like the first half of this sentence reads weirdly and could maybe be rethought tho. Won't block on it but hm.

Reply:
I don't like it much either but I'm bad at writing this sort of thing, so all I did was clean up what was there originally (in particular fixing the actionlinks), but also a bit of spelling and making the last sentence less gendered.

Any better writing suggestions welcome.

Reply:
Maybe something like this? The wording is mostly fine, just reads a bit odd grammatically.

```suggestion
			<p>This analyser attempts to find just the right things to turn you into a fearsome tank that will show no <ActionLink {...ACTIONS.CLEMENCY}/> to your enemies while being the true <ActionLink {...ACTIONS.HOLY_SPIRIT}/> of the party.
```